conf/messages.en

```diff
@@ -383,6 +383,7 @@ organisationDetails.type.ScottishLimitedPartnership = Scottish limited partnersh
 
 organisationDetails.type.GroupNominatedPartnership = Limited liability partnership
 organisationDetails.type.hint = Register for Plastic Packaging Tax if you are a charitable incorporated organisation (CIO), a trust, an overseas organisation without a UK establishment, or a public body.
+organisationDetails.type.href = https://www.tax.service.gov.uk/submissions/new-form/register-for-plastic-packaging-tax
```

Comment:
url currently returns a 404 - is that expected?

Reply:
I don't know @PSouthern 🤔 . It was the url I was passed from Rebecca.

Reply:
that url will only go live on 31 March